p2p/simulations/examples/p2psim.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
This script is specific to the example though (it uses the `ping-pong` service).

I plan to fully document all of this code before merging, so there will be `p2p/simulations/doc.go` and `p2p/simulations/README.md` with examples.